securedrop/manage.py

```diff
@@ -159,6 +159,12 @@ def _add_user(is_admin=False):
             otp_secret = raw_input(
                 "Please configure this user's YubiKey and enter the secret: ")
             if otp_secret:
+                tmp_str = otp_secret.replace(" ", "")
+                if len(tmp_str) != 40:
+                    print("The length of the secret is not correct. "
+                          "Try again.")
```

Comment:
It would be very useful to mention the length of the secret provided and the expected length.

Reply:
Will add that.